templates/500.html

```diff
@@ -1,39 +1,15 @@
-{% extends "base.html" %}
+{% extends "base.html"%}
 
-{% block title %}Internal Server Error - Online{% endblock title %}
 
 {% block content %}
-    <section id="error">
-        <div class="container">
-            <div class="row">
-                <div class="col-md-12">
-                    <div class="page-header">
-                        <h2 id="error-heading">500 Error: Internal server error</h2>
-                    </div>
-                </div>
-            </div>
-            <div class="row">
-                <div class="col">
-                    <p>Uff da, det har oppstått en feil.</p>
-                    <p>Vi blir varslet om dette automatisk, men vi setter pris på om du vil legge igjen en beskjed for hvordan du havnet her. </p>
-                </div>
-            </div>
-            <div class="error-items centered">
-                <form action="" method="post" >
-                    <div class="row">
-                        {% csrf_token %}
-                        <div class="col-md-12">
-                            {% for field in error_form %}
-                                <b>{{ field.label }}</b>
-                                <div class="row ">
-                                    {{ field }}
-                                </div>
-                            {% endfor %}
-                        </div>
-                        <button class="btn btn-success" type="submit">Send mail</button>
-                    </div>
-                </form>
-            </div>
-        </div>
-    </section>
+<script src="https://browser.sentry-cdn.com/5.15.4/bundle.min.js" integrity="sha384-Nrg+xiw+qRl3grVrxJtWazjeZmUwoSt0FAVsbthlJ5OMpx0G08bqIq3b/v0hPjhB" crossorigin="anonymous"></script>
+
+{% if sentry_event_id %}
+<section>
```

Comment:
I think you forgot to close this `<section>`? Also I would _love_ if you just added something like `height: 90vh` or some alternative to `<main>` perhaps just for this page, so that the background looks a tiny bit better, but great work so far!